### ISSUE TITLE
🐛 BUG: Inline images throws error on build

### ISSUE THREAD

**Issue by Artemis69** [snowpackjs/astro#1102]
### What package manager are you using?

pnpm

### What operating system are you using?

Linux (WSL 2)

### Describe the Bug

I am using `0.19.0-next.2` astro version and `16.2.0` node version

While building astro will try to find images like below
```html
<img 
  src="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='24' height='24' fill='none'%3E%3Cdefs/%3E%3Cpath stroke='currentColor' stroke-linecap='round' stroke-linejoin='round' stroke-width='1.5' d='M17.25 12v-2a5.25 5.25 0 10-10.5 0v2l-2 4.25h14.5l-2-4.25zM9 16.75s0 2.5 3 2.5 3-2.5 3-2.5'/%3E%3C/svg%3E" 
  alt="bell"
>
```
This will thow a error, but only on build

### Steps to Reproduce

1. `npm init astro` using template Starter Kit (Generic)
2. Insert svg image into img`s src
3. `pnpm build`
4. Error:
[build] file:///home/artem/Documents/wawawa/src/pages/index.astro: could not find "svg%3E"
Astro should not try to search for such pictures sources

### Link to Minimal Reproducible Example (Optional)

https://github.com/Artemis69/bug-astro-reproduce

**Comment by drwpow** [comment]
This is definitely a bug. This should be easily-fixable by just adding some code [to our image scanner](https://github.com/snowpackjs/astro/blob/0f0cc2b9d83ffeee125b85c95d2d51080f21074d/packages/astro/src/build.ts#L310) that detects `data:image/…` as the start of a string and ignores it. Would anyone like to open this PR?

**Comment by jonathantneal** [comment]
This looks to have been fixed in https://github.com/snowpackjs/astro/commit/cb43ca440bc74fa3de3547f78c730461177223f0

**Comment by drwpow** [comment]
Ah @jonathantneal. That would make this a duplicate of #1052, then. Closing, as that’s been merged to `main` (and I believe it’s already been released in `0.18.10`)
